samples/Azure.ResourceManager.Sample/Generated/Extensions/MockableSampleSubscriptionResource.cs

```diff
@@ -110,9 +110,9 @@ public virtual VirtualMachineExtensionImageCollection GetVirtualMachineExtension
         /// </list>
         /// </summary>
         /// <param name="location"> The name of a supported Azure region. </param>
-        /// <param name="publisherName"> The String to use. </param>
-        /// <param name="type"> The String to use. </param>
-        /// <param name="version"> The String to use. </param>
+        /// <param name="publisherName"> The string to use. </param>
```

Comment:
I think we should use crefs here so that when it links to a non primitive you have a nice reference.